### ISSUE TITLE
Config.json does not disable video recording

### ISSUE THREAD

**Issue by duhon** [sitespeedio/sitespeed.io#2692]
Command:
```
docker run --rm -v $(pwd):/sitespeed.io sitespeedio/sitespeed.io:10.2.0 --config config.json --debug https://www.sitespeed.io/
```
Config:
```
{
  "browsertime": {
    "verification-config": "test",
    "iterations": 1,
    "visualMetrics": false,
    "video": false
  },
  "visualMetrics": false,
  "video": false
}
```
Result:
```
Running with options: {
  "_": [
    "https://www.sitespeed.io/"
  ],
  "config": "config.json",
  "debug": 1,
  "verbose": 1,
  "browsertime": {
    "video": true,
    "visualMetrics": true,
    "xvfb": "true",
    "verification-config": "test",
.....
[2019-10-10 13:45:40] INFO: [browsertime] Running tests using Chrome - 1 iteration(s)
[2019-10-10 13:45:40] INFO: [browsertime.command.measure] Testing url https://www.sitespeed.io/ iteration 1
[2019-10-10 13:45:40] DEBUG: [browsertime.video] Add orange color
[2019-10-10 13:45:40] DEBUG: [browsertime.video] Started ffmpeg with -hide_banner -video_size 1366x642 -f x11grab -framerate 30 -probesize 10M -y -draw_mouse 0 -i :99.0+0,66 -codec:v libx264rgb -crf 0 -preset ultrafast -vf pad=ceil(iw/2)*2:ceil(ih/2)*2 /sitespeed.io/sitespeed-result/www.sitespeed.io/2019-10-10-13-45-39/x11-tmp.mp4
```

**How do I turn off the video recording through the config?**
If I run with the options `--visualMetrics=false --video=false`, then the video will not be recorded

**Comment by duhon** [comment]
It looks like we can convert a config.json into env variables before starting a CLI command and before other env vars

**Comment by soulgalore** [comment]
Yes. I think there's simpler one too, but I need to to test it. The ENV is set in the Docker file, and we could probably remove it and then if the video is not set + running in Docker automagically set it. Then it will work as before except that config.json will rule.

**Comment by soulgalore** [comment]
This is fixed in 11.0.0.
